### PR TITLE
Added WT60-B, WT60-BX (wilba.tech THERMAL)

### PR DESCRIPTION
**Pull request by Wilba6582**
Added WT60-B, WT60-BX (wilba.tech THERMAL)

Last addition to `wt_rgb_backlight.c` before I refactor the `#ifdef` bloat.

Please don't make me clang-format this, it wrecks all the array initialization 😢

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/wilba_tech/wt60_b/readme.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# wilba.tech WT60-B
+
+WT60-B is a keyboard PCB supporting 60% layout. [More info at wilba.tech](https://wilba.tech/)
+
+First produced as WT60-B THERMAL for wilba.tech Thermal.
+
+WT60-B is the hotswap variant PCB.
+WT60-BX is the solderable variant PCB with split backspace and stepped Caps Lock support.
+
+Keyboard Maintainer: [Wilba6582](https://github.com/Wilba6582)
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt60_b/readme.md]:
```suggestion
* Keyboard Maintainer: [Wilba6582](https://github.com/Wilba6582)
```

File: keyboards/wilba_tech/wt60_b/readme.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+Hardware Supported: wilba.tech WT60-B
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt60_b/readme.md]:
```suggestion
* Hardware Supported: wilba.tech WT60-B
```

File: keyboards/wilba_tech/wt60_b/readme.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+Hardware Availability: Custom keyboard group buys
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt60_b/readme.md]:
```suggestion
* Hardware Availability: Custom keyboard group buys
```

File: keyboards/wilba_tech/wt60_b/info.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+    "keyboard_name": "wilba.tech WT60-B",
+    "url": "https://wilba.tech",
+    "maintainer": "Wilba",
+    "bootloader": "atmel-dfu",
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt60_b/info.json]: A bit of a nitpick, but the `bootloader` key for `info.json` files isn't used anymore.

```suggestion
```

File: keyboards/wilba_tech/wt60_bx/info.json [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{
+    "keyboard_name": "wilba.tech WT60-BX",
+    "url": "https://wilba.tech",
+    "maintainer": "Wilba",
+    "bootloader": "atmel-dfu",
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt60_bx/info.json]:
```suggestion
```

File: keyboards/wilba_tech/wt60_bx/readme.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# wilba.tech WT60-BX
+
+WT60-BX is a keyboard PCB supporting 60% layout. [More info at wilba.tech](https://wilba.tech/)
+
+First produced as WT60-BX THERMAL for wilba.tech Thermal.
+
+WT60-B is the hotswap variant PCB.
+WT60-BX is the solderable variant PCB with split backspace and stepped Caps Lock support.
+
+Keyboard Maintainer: [Wilba6582](https://github.com/Wilba6582)
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt60_bx/readme.md]:
```suggestion
* Keyboard Maintainer: [Wilba6582](https://github.com/Wilba6582)
```

File: keyboards/wilba_tech/wt60_bx/readme.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+Hardware Supported: wilba.tech WT60-BX
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt60_bx/readme.md]:
```suggestion
* Hardware Supported: wilba.tech WT60-BX
```

File: keyboards/wilba_tech/wt60_bx/readme.md [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+Hardware Availability: Custom keyboard group buys
```
Comment (noroadsleft) [inline keyboards/wilba_tech/wt60_bx/readme.md]:
```suggestion
* Hardware Availability: Custom keyboard group buys
```

**Comment by Wilba6582** [comment]
@noroadsleft changes made as suggested, updated all the `info.json` as well.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**